scenarioo-client/app/shared/searchEngineStatus.service.js

```diff
@@ -0,0 +1,38 @@
+/* scenarioo-client
+ * Copyright (C) 2014, scenarioo.org Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+angular.module('scenarioo.services').factory('SearchEngineStatusService', function (SelectedBranchAndBuildService, FullTextSearchService) {
+    return {
+        isEngineRunning: function() {
+            var selected = SelectedBranchAndBuildService.selected();
+
+            var promise = FullTextSearchService.checkStatus({
+                    buildName: selected.build,
+                    branchName: selected.branch
+                }
```

Comment:
I am removing this and am also changing the behavior of the REST service to return a boolean instead of using the HTTP status to indicate whether the service is running or not (see other comment further down).